src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.form {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label::before {
+  content: "";
+  position: absolute;
+  background-image: url("images/Search.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  z-index: 1;
+}
+
+.label--small::before {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 400;
+  color: #3d4e61;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)